File: server/src/main/java/au/com/codeka/warworlds/server/admin/handlers/SectorsHandler.java
package au.com.codeka.warworlds.server.admin.handlers;

import au.com.codeka.warworlds.server.handlers.RequestException;

/**
 * This handler is for /admin/sector, and allows us to debug some of the sector handling code.
 */
public class SectorsHandler extends AdminHandler {
  @Override
  public void get() throws RequestException {
    render("sectors/index.html", null);
  }
}

File: server/src/main/kotlin/au/com/codeka/warworlds/server/admin/handlers/SectorsHandler.kt
package au.com.codeka.warworlds.server.admin.handlers

import au.com.codeka.warworlds.server.handlers.RequestException

/**
 * This handler is for /admin/sector, and allows us to debug some of the sector handling code.
 */
class SectorsHandler : AdminHandler() {
  public override fun get() {
    render("sectors/index.html", null)
  }
}